app/javascript/shared/components/tags.jsx

```diff
@@ -439,7 +440,7 @@ class Tags extends Component {
           placeholder={`${maxTags} tags max, comma separated, no spaces or special characters`}
           autoComplete="off"
           value={defaultValue}
-          onInput={this.handleInput}
+          onInput={debounce(this.handleInput, 300)}
```

Comment:
can we move it in the initializer like we did in https://github.com/thepracticaldev/dev.to/pull/6322 - this would create a new anonymous function each time we type a key, kinda defeating the purpose of having `this.handleInput`